odpf/assets/common/v1beta2/resource.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package odpf.assets.common.v1beta2;
+
+option java_package = "io.odpf.assets.common";
+option java_outer_classname = "ResourceProto";
+option go_package = "github.com/odpf/proton/assets/common/v1beta2;commonv1beta2";
+
+
+// Resource is a generic resource that represents a file or other resource.
+// It can be a table, job, user or group.
+message Resource {
```

Comment:
have flattened `Asset.Resource.*` to `Asset.*`, but could not delete the Resource.proto because it is being used in lineage.proto

Reply:
We can move resource to `lineage.proto` since it is the only place it is used.